SavedGoogleDrive.csv

```diff
@@ -0,0 +1,53 @@
+id,MSB,LSB,name,comments
+0x500,0,0,Raw Throttle Value 1,
+0x500,1,1,Throttle Value,
+0x500,2,3,Brake Value,
+0x700,0,1,FL Wheel Speed,Wheel speed in RPM
+0x700,2,3,FR Wheel Speed,
+0x701,0,1,RL Wheel Speed,
+0x701,2,3,RR Wheel Speed,
+0x710,0,3,FL Tire Temp,Temperature in deg C
+0x710,4,7,FR Tire Temp,
+0x711,0,3,RL Tire Temp,
+0x711,4,7,RR Tire Temp,
+0x720,0,5,Coolant Temp,
+0x720,6,7,Coolant Flow,
+0x730,0,1,FL Shock Pot,
+0x730,2,3,FR Shock Pot,
+0x731,0,1,RL Shock Pot,
+0x731,2,3,RR Shock Pot,
+0x740,0,1,ARBDropLink_Left,
+0x740,2,3,ARBDropLink_Right,
+0x740,4,5,FL ARB Droplink,
+0x740,6,7,FR ARB Droplink,
```

Comment:
Why are the droplinks in here twice (lines 19-22)? Or do they require multiple bytes? Should either be removed if duplicate or named the same thing. Or I'm misunderstanding what's going on here.

Reply:
This was automatically generated by a script based on what is in here: https://docs.google.com/spreadsheets/d/195Y2cf9C2mrA6QCLuLJv3NHsCqTLi-3Wd07q4T5rXuM/edit#gid=64209652

We can change the spreadsheet and it will reflect once you run the script again.